project.clj

```diff
@@ -1,7 +1,7 @@
 (def ks-version "1.2.0")
 (def tk-version "1.2.0")
 
-(defproject puppetlabs/trapperkeeper-metrics "0.1.3-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-metrics "0.2.0-SNAPSHOT"
```

Comment:
Edit: A `:patch` release, ideally we'd be able to do a `:minor` release from the job without bumping the version here but that sounds like it would cause the problem you're describing @fpringvaldsen